src/ansys/fluent/core/session_meshing.py

```diff
@@ -0,0 +1,202 @@
+"""Module containing class encapsulating Fluent connection."""
+import grpc
+
+from ansys.fluent.core.services.datamodel_se import PyMenuGeneric
+from ansys.fluent.core.services.datamodel_tui import TUIMenuGeneric
+from ansys.fluent.core.session import (
+    _CODEGEN_MSG_TUI,
+    BaseSession,
+    parse_server_info_file,
+)
+from ansys.fluent.core.session_solver import Solver
+from ansys.fluent.core.utils.logging import LOG
+
+_CODEGEN_MSG_DATAMODEL = (
+    "Currently calling the datamodel API in a generic manner. "
+    "Please run `python codegen/allapigen.py` from the top-level pyfluent "
+    "directory to generate the local datamodel API classes."
+)
+
+
+class Meshing(BaseSession):
```

Comment:
Can this be derived from PureMeshing?

Reply:
Sure, that can be done. For now, only the structure has been made. Later, Meshing can be derived from PureMeshing and solver from Solver-Lite and only the additional methods to be defied in the Meshing and Solver. Should that be fine or shall we make the changes now only?

Reply:
Yeah, it is fine to make this change later.

Reply:
@prmukherj temporarily unresolving this conversation to say that I was going to comment similarly to @mkundu1. *There are various solutions here*, but in any case the duplication should be eliminated. It doesn't have to be done now, but it would be preferable to have it cleaned up in the near term.
https://github.com/pyansys/pyfluent/issues/699
Please resolve the conversation to acknowledge, thanks.